<a href="https://colab.research.google.com/github/May-o-nnaise/CPSMA-3933-MS/blob/main/DataVizQuiz8.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#Quiz 8

In [ ]:
from google.colab import auth
auth.authenticate_user()
print('Authenticated')

Authenticated


##3.

In [ ]:
%%bigquery --project dataviz-488505
SELECT
  COUNT(*) as total_rows
FROM `bigquery-public-data.austin_bikeshare.bikeshare_trips`

Query is running:   0%|          |

Downloading:   0%|          |

total_rows
0     2271152

This took a long time to make work and I did end up having to use my credit card. Expect an invoice sometime next month!

I also couldn't get my "project" to import the entire bigquery thing, at least i dont think so anyways Im still kinda confused what I am looking at. I think I could only get it to get the austin bikeshare and all of the tables with that.

##4.

In [ ]:
%%bigquery --project dataviz-488505
SELECT
  MAX(duration_minutes)
FROM `bigquery-public-data.austin_bikeshare.bikeshare_trips`



Query is running:   0%|          |

Downloading:   0%|          |

f0_
0  34238

Am I tweaking? someone had a bike ride that was 34238 minutes? What is that like 20 days?

##5.

In [ ]:
%%bigquery --project dataviz-488505
SELECT
  AVG(duration_minutes)
FROM `bigquery-public-data.austin_bikeshare.bikeshare_trips`

Query is running:   0%|          |

Downloading:   0%|          |

f0_
0  28.719178

Average bike ride looks a little less than 30 minutes which is much more believable than the 34000+ minute trip